packages/fxa-settings/src/lib/gql-key-stretch-upgrade.ts

```diff
@@ -186,13 +185,24 @@ export class GqlKeyStretchUpgrade {
         passwordChangeToken,
       };
     } catch (error) {
-      // If the user enters the wrong password, they will see an invalid password error.
-      // Otherwise something has going wrong and we should show a general error.
+      const errno = getHandledError(error).error.errno;
+
+      if (errno === 104) {
+        // Session not verified. Trying again later.
+        console.info('Account not verified. Try upgrade later.');
+      } else if (errno === 138) {
+        // Account not verified. Trying again later.
```

Comment:
Same as above, comment necessary? If so swap to 
```suggestion
        // Session not verified. Trying again later.
```